extensions/amp-story/0.1/progress-bar.js

```diff
@@ -144,9 +155,15 @@ export class ProgressBar {
         `.i-amphtml-story-page-progress-bar:nth-child(${nthChildIndex}) ` +
         '.i-amphtml-story-page-progress-value');
     this.vsync_.mutate(() => {
+      let transition = 'none';
+      if (withTransition) {
+        // Using an eased transition only if filling the bar to 0 or 1.
+        transition =
+            progress === ~~progress ? TRANSITION_EASE : TRANSITION_LINEAR;
```

Comment:
May be better to just be explicit with the condition: `progress === 0 || progress === 1`

Reply:
Done!